servicereportpkg/validate/plugins/spyre.py

```diff
@@ -19,6 +20,7 @@
 from servicereportpkg.utils import is_package_installed
 from servicereportpkg.check import ConfigurationFileCheck
 from servicereportpkg.utils import is_read_write_to_all_users
```

Comment:
This function is not longer used. So lets remove the import. More info info the comment below.